token/program-2022-test/tests/transfer_fee.rs

```diff
@@ -793,7 +799,14 @@ async fn transfer_checked() {
 
     // fail unchecked always
     let error = token
-        .transfer_unchecked(&alice_account, &bob_account, &alice, maximum_fee)
+        .transfer(
+            &alice_account,
+            &alice_account,
```

Comment:
nit: Let's keep this as `&bob_account`

Reply:
aaah thank you, this was a legitimate mistake from overzealous copy-paste while fixing the many test invocations